libs/blocks/media/media.js

```diff
@@ -34,6 +34,8 @@ function decorateQr(el) {
   const googlePlay = text.children[(text.children.length - 2)]?.querySelector('a');
   const qrImage = text.children[(text.children.length - 3)];
   if (!qrImage || !appStore || !googlePlay) return;
+  appStore.parentElement.classList.add('qr-button-container');
+  googlePlay.parentElement.classList.add('qr-button-container');
```

Comment:
Nit: more of a subjective change, but I would make it an one liner
```suggestion
[appStore, googlePlay].forEach(el => el.parentElement.classList.add('qr-button-container'));
```

Reply:
Added 👍